### ISSUE TITLE
Krona generation fails:  all seqs are orphans?

### ISSUE THREAD

**Issue by wolfgangrumpf** [khyox/recentrifuge#14]
I am using kraken2 output in recentrifuge and am not getting the results I expected:

**Command**:
$ rcf -n ../taxdump/ --debug -k .

Input file: (see attached)
[krakenout.krk.zip](https://github.com/khyox/recentrifuge/files/3178880/krakenout.krk.zip)

**Output**:
=-= /gpfs0/home/gdlauberlab/rwr002/.local/bin/rcf =-= v0.28.8 - Apr 2019 =-= by Jose Manuel Martí =-=

INFO: Debugging mode activated
INFO: Active parameters:
	nodespath = ../taxdump/
	kraken = ['.']
	extra = FULL
	controls = 0
	scoring = SHEL
	summary = add
	debug = True
Kraken .krk files to analyze: ['krakenout.krk']
Loading NCBI nodes... OK!
Loading NCBI names... OK!
Building dict of parent to children taxa... OK!

Please, wait, processing files in parallel...

Processing sample krakenout.krk ...
Loading output file krakenout.krk... OK!
  Seqs read: 15_884	[25.51 Mnt]
  Seqs clas: 15_884	(0.00% unclassified)
  Seqs pass: 15_884	(0.00% rejected)
  Scores SHEL: min = 35.0, max = 35.0, avr = 35.0
  Coverage(%): min = 0.0, max = 0.0, avr = 0.0
  Read length: min = 460 nt, max = 6.08 knt, avr = 1.60 knt
  TaxIds: by classifier = 186, by filter = 186
Building from raw data with mintaxa = 4 ...
  Building ontology tree with all-in-1... OK!
  Check for more seqs lost ([in/ex]clude affects)...
  Info: 15884 additional seqs discarded (100.000% of accepted)
  Checking taxid loss (orphans)...   Warning! Orphan taxid Streptococcus dysgalactiae subsp. equisimilis AC-2713 (taxid 759913)
  Warning! Orphan taxid Shigella boydii (taxid 621)
  Warning! Orphan taxid Staphylococcus epidermidis (taxid 1282)
  Warning! Orphan taxid Pseudomonas koreensis (taxid 198620)
  Warning! Orphan taxid Pseudomonas aeruginosa group (taxid 136841)
  Warning! Orphan taxid Pseudomonas aeruginosa B136-33 (taxid 1280938)
  Warning! Orphan taxid Lactobacillus (taxid 1578)
  Warning! Orphan taxid Shigella flexneri 2002017 (taxid 591020)
  Warning! Orphan taxid Streptococcus pyogenes MGAS15252 (taxid 798300)
  Warning! Orphan taxid Acinetobacter sp. FDAARGOS_493 (taxid 2420300)
  Warning! Orphan taxid Bacillus cereus group (taxid 86661)
  Warning! Orphan taxid Enterobacter bugandensis (taxid 881260)
  Warning! Orphan taxid Pseudomonadaceae (taxid 135621)
  Warning! Orphan taxid Escherichia coli P12b (taxid 910348)
  Warning! Orphan taxid Neisseria meningitidis M7124 (taxid 1095685)
  Warning! Orphan taxid Clostridium botulinum (taxid 1491)
  Warning! Orphan taxid Lactobacillus sp. HSLZ-75 (taxid 1720083)
  Warning! Orphan taxid Acinetobacter radioresistens (taxid 40216)
  Warning! Orphan taxid Pseudomonas fluorescens (taxid 294)
  Warning! Orphan taxid Yersinia pestis Angola (taxid 349746)
  Warning! Orphan taxid Escherichia coli ABU 83972 (taxid 655817)
  Warning! Orphan taxid Staphylococcus sp. AntiMn-1 (taxid 1715860)
  Warning! Orphan taxid Escherichia coli Nissle 1917 (taxid 316435)
  Warning! Orphan taxid Lactobacillus delbrueckii subsp. delbrueckii (taxid 83684)
  Warning! Orphan taxid Bacteria (taxid 2)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Napoli (taxid 1151001)
  Warning! Orphan taxid Clostridium perfringens str. 13 (taxid 195102)
  Warning! Orphan taxid Streptococcus mutans (taxid 1309)
  Warning! Orphan taxid Streptococcus iniae (taxid 1346)
  Warning! Orphan taxid Clostridium baratii (taxid 1561)
  Warning! Orphan taxid Edwardsiella ictaluri (taxid 67780)
  Warning! Orphan taxid Bacillus thuringiensis serovar finitimus YBT-020 (taxid 930170)
  Warning! Orphan taxid Lactobacillus delbrueckii subsp. lactis (taxid 29397)
  Warning! Orphan taxid Bacteroides vulgatus (taxid 821)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Montevideo str. USDA-ARS-USMARC-1901 (taxid 1454598)
  Warning! Orphan taxid Acinetobacter baumannii TYTH-1 (taxid 1100841)
  Warning! Orphan taxid Streptococcus suis (taxid 1307)
  Warning! Orphan taxid Staphylococcus argenteus (taxid 985002)
  Warning! Orphan taxid Actinomyces pacaensis (taxid 1852377)
  Warning! Orphan taxid Bacillus cereus D17 (taxid 1454382)
  Warning! Orphan taxid Pseudomonas protegens (taxid 380021)
  Warning! Orphan taxid Klebsiella michiganensis (taxid 1134687)
  Warning! Orphan taxid Escherichia coli UMNF18 (taxid 1050617)
  Warning! Orphan taxid Staphylococcus haemolyticus (taxid 1283)
  Warning! Orphan taxid Cutibacterium acnes (taxid 1747)
  Warning! Orphan taxid Streptococcus parauberis NCFD 2020 (taxid 873447)
  Warning! Orphan taxid Staphylococcus aureus (taxid 1280)
  Warning! Orphan taxid Streptococcus pyogenes (taxid 1314)
  Warning! Orphan taxid Enterococcaceae (taxid 81852)
  Warning! Orphan taxid Pseudomonas (taxid 286)
  Warning! Orphan taxid Clostridium beijerinckii NCIMB 8052 (taxid 290402)
  Warning! Orphan taxid Staphylococcus aureus subsp. aureus TCH60 (taxid 548473)
  Warning! Orphan taxid Acinetobacter calcoaceticus/baumannii complex (taxid 909768)
  Warning! Orphan taxid Pseudomonas litoralis (taxid 797277)
  Warning! Orphan taxid Lactobacillus reuteri (taxid 1598)
  Warning! Orphan taxid Lelliottia amnigena (taxid 61646)
  Warning! Orphan taxid Rhodobacter sphaeroides (taxid 1063)
  Warning! Orphan taxid Escherichia coli O104:H4 (taxid 1038927)
  Warning! Orphan taxid Acinetobacter sp. SWBY1 (taxid 2079596)
  Warning! Orphan taxid Porphyromonas gingivalis (taxid 837)
  Warning! Orphan taxid Lactobacillus helveticus (taxid 1587)
  Warning! Orphan taxid Clostridium bornimense (taxid 1216932)
  Warning! Orphan taxid Bacillus anthracis str. Vollum (taxid 261591)
  Warning! Orphan taxid Streptococcus sp. FDAARGOS_522 (taxid 2420310)
  Warning! Orphan taxid Streptococcus halotolerans (taxid 1814128)
  Warning! Orphan taxid Clostridium autoethanogenum DSM 10061 (taxid 1341692)
  Warning! Orphan taxid Pseudomonas aeruginosa LESB65 (taxid 1408273)
  Warning! Orphan taxid Streptococcus (taxid 1301)
  Warning! Orphan taxid Staphylococcus epidermidis ATCC 12228 (taxid 176280)
  Warning! Orphan taxid Clostridium beijerinckii NRRL B-598 (taxid 1428454)
  Warning! Orphan taxid Bifidobacterium adolescentis (taxid 1680)
  Warning! Orphan taxid Escherichia coli 536 (taxid 362663)
  Warning! Orphan taxid Streptococcus agalactiae GD201008-001 (taxid 1203670)
  Warning! Orphan taxid Escherichia coli ACN001 (taxid 1311757)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Senftenberg (taxid 28150)
  Warning! Orphan taxid Bifidobacterium adolescentis ATCC 15703 (taxid 367928)
  Warning! Orphan taxid Klebsiella pneumoniae (taxid 573)
  Warning! Orphan taxid Staphylococcus simulans (taxid 1286)
  Warning! Orphan taxid Candidatus Annandia adelgestsuga (taxid 1302411)
  Warning! Orphan taxid Clostridium saccharobutylicum (taxid 169679)
  Warning! Orphan taxid Streptococcus pneumoniae SPN032672 (taxid 869311)
  Warning! Orphan taxid Clostridium sp. JN-9 (taxid 2507159)
  Warning! Orphan taxid Acinetobacter pittii PHEA-2 (taxid 871585)
  Warning! Orphan taxid Streptococcus suis T15 (taxid 1340847)
  Warning! Orphan taxid Escherichia coli (taxid 562)
  Warning! Orphan taxid Staphylococcus pasteuri (taxid 45972)
  Warning! Orphan taxid Lactobacillus plantarum (taxid 1590)
  Warning! Orphan taxid Salmonella enterica subsp. enterica (taxid 59201)
  Warning! Orphan taxid Acinetobacter baumannii ZW85-1 (taxid 1400867)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Typhimurium (taxid 90371)
  Warning! Orphan taxid Acinetobacter baumannii PKAB07 (taxid 1413216)
  Warning! Orphan taxid Acinetobacter baumannii (taxid 470)
  Warning! Orphan taxid Streptococcus pluranimalium (taxid 82348)
  Warning! Orphan taxid Pseudomonas aeruginosa LES431 (taxid 1408272)
  Warning! Orphan taxid Streptococcus sp. Z15 (taxid 2173853)
  Warning! Orphan taxid Rhodobacter sphaeroides 2.4.1 (taxid 272943)
  Warning! Orphan taxid Shigella dysenteriae (taxid 622)
  Warning! Orphan taxid Clostridium cellulovorans 743B (taxid 573061)
  Warning! Orphan taxid Streptococcus equi (taxid 1336)
  Warning! Orphan taxid Clostridium acetobutylicum (taxid 1488)
  Warning! Orphan taxid Acinetobacter nosocomialis (taxid 106654)
  Warning! Orphan taxid Streptococcus intermedius C270 (taxid 862966)
  Warning! Orphan taxid Rhodobacter sphaeroides ATCC 17029 (taxid 349101)
  Warning! Orphan taxid Aerococcus (taxid 1375)
  Warning! Orphan taxid Enterobacteriaceae (taxid 543)
  Warning! Orphan taxid Cronobacter sakazakii SP291 (taxid 956149)
  Warning! Orphan taxid Acinetobacter oleivorans DR1 (taxid 436717)
  Warning! Orphan taxid Enterococcus faecalis (taxid 1351)
  Warning! Orphan taxid Clostridium sp. MF28 (taxid 1702238)
  Warning! Orphan taxid Bacillus (taxid 1386)
  Warning! Orphan taxid Neisseria meningitidis WUE 2594 (taxid 942513)
  Warning! Orphan taxid Bacillus thuringiensis MC28 (taxid 1195464)
  Warning! Orphan taxid Staphylococcus succinus (taxid 61015)
  Warning! Orphan taxid Streptococcus dysgalactiae subsp. equisimilis (taxid 119602)
  Warning! Orphan taxid Acinetobacter bereziniae (taxid 106648)
  Warning! Orphan taxid Enterobacter cloacae (taxid 550)
  Warning! Orphan taxid Streptococcus pyogenes str. Manfredo (taxid 160491)
  Warning! Orphan taxid Acinetobacter venetianus VE-C3 (taxid 1197884)
  Warning! Orphan taxid Streptococcus sp. FDAARGOS_520 (taxid 2420308)
  Warning! Orphan taxid Acinetobacter (taxid 469)
  Warning! Orphan taxid Staphylococcus schleiferi (taxid 1295)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Bareilly str. CFSAN000191 (taxid 1173457)
  Warning! Orphan taxid Escherichia coli LY180 (taxid 1335916)
  Warning! Orphan taxid Neisseria meningitidis M04-240196 (taxid 935593)
  Warning! Orphan taxid Bacillus cereus (taxid 1396)
  Warning! Orphan taxid Lactobacillus gasseri ATCC 33323 = JCM 1131 (taxid 324831)
  Warning! Orphan taxid Bacillus anthracis (taxid 1392)
  Warning! Orphan taxid Lactobacillus lindneri (taxid 53444)
  Warning! Orphan taxid Enterobacter asburiae (taxid 61645)
  Warning! Orphan taxid Bacillus thuringiensis (taxid 1428)
  Warning! Orphan taxid Clostridium beijerinckii (taxid 1520)
  Warning! Orphan taxid Clostridium sp. CT4 (taxid 2320868)
  Warning! Orphan taxid Clostridium estertheticum subsp. estertheticum (taxid 1552)
  Warning! Orphan taxid Staphylococcus aureus subsp. aureus NCTC 8325 (taxid 93061)
  Warning! Orphan taxid Staphylococcus auricularis (taxid 29379)
  Warning! Orphan taxid Staphylococcus caprae (taxid 29380)
  Warning! Orphan taxid Staphylococcus aureus 08BA02176 (taxid 1229492)
  Warning! Orphan taxid Staphylococcus muscae (taxid 1294)
  Warning! Orphan taxid Bacteroides vulgatus ATCC 8482 (taxid 435590)
  Warning! Orphan taxid Helicobacter pylori OK113 (taxid 1248725)
  Warning! Orphan taxid Deinococcus radiodurans R1 (taxid 243230)
  Warning! Orphan taxid Enterobacterales (taxid 91347)
  Warning! Orphan taxid Pseudomonas entomophila (taxid 312306)
  Warning! Orphan taxid Clostridium perfringens (taxid 1502)
  Warning! Orphan taxid Clostridium drakei (taxid 332101)
  Warning! Orphan taxid Escherichia coli APEC IMT5155 (taxid 1329907)
  Warning! Orphan taxid Helicobacter pylori (taxid 210)
  Warning! Orphan taxid Clostridium argentinense (taxid 29341)
  Warning! Orphan taxid Staphylococcus (taxid 1279)
  Warning! Orphan taxid Pseudomonas putida (taxid 303)
  Warning! Orphan taxid Staphylococcus capitis (taxid 29388)
  Warning! Orphan taxid Acinetobacter haemolyticus (taxid 29430)
  Warning! Orphan taxid Escherichia coli APEC O1 (taxid 405955)
  Warning! Orphan taxid Escherichia coli O104:H4 str. 2011C-3493 (taxid 1133852)
  Warning! Orphan taxid Pseudomonas aeruginosa PA7 (taxid 381754)
  Warning! Orphan taxid Lactobacillus helveticus H10 (taxid 767462)
  Warning! Orphan taxid Rhodobacter (taxid 1060)
  Warning! Orphan taxid Staphylococcus condimenti (taxid 70255)
  Warning! Orphan taxid Enterococcus faecalis ARO1/DG (taxid 565651)
  Warning! Orphan taxid Acinetobacter johnsonii (taxid 40214)
  Warning! Orphan taxid Pseudomonas chlororaphis (taxid 587753)
  Warning! Orphan taxid Streptococcus agalactiae ILRI005 (taxid 1309807)
  Warning! Orphan taxid Bacillus thuringiensis serovar kurstaki str. HD-1 (taxid 1261129)
  Warning! Orphan taxid Staphylococcus aureus subsp. aureus MSSA476 (taxid 282459)
  Warning! Orphan taxid Salmonella enterica subsp. enterica serovar Heidelberg str. B182 (taxid 1160717)
  Warning! Orphan taxid Bacillus cereus biovar anthracis str. CI (taxid 637380)
  Warning! Orphan taxid Staphylococcus aureus subsp. aureus (taxid 46170)
  Warning! Orphan taxid Streptococcus mutans UA159 (taxid 210007)
  Warning! Orphan taxid Streptococcus agalactiae (taxid 1311)
  Warning! Orphan taxid Enterobacter ludwigii (taxid 299767)
  Warning! Orphan taxid Lactobacillus johnsonii (taxid 33959)
  Warning! Orphan taxid Acinetobacter baumannii BJAB0868 (taxid 1096997)
  Warning! Orphan taxid Clostridium (taxid 1485)
  Warning! Orphan taxid Staphylococcus stepanovicii (taxid 643214)
  Warning! Orphan taxid Morganella morganii (taxid 582)
  Warning! Orphan taxid Lactobacillus murinus (taxid 1622)
  Warning! Orphan taxid Pseudomonas xinjiangensis (taxid 487184)
  Warning! Orphan taxid Escherichia coli O157:H7 str. EDL933 (taxid 155864)
  Warning! Orphan taxid Pseudomonas aeruginosa (taxid 287)
  Warning! Orphan taxid Staphylococcus lugdunensis (taxid 28035)
  Warning! Orphan taxid Bacillus cereus ATCC 10987 (taxid 222523)
  Warning! Orphan taxid Neisseria meningitidis (taxid 487)
  Warning! Orphan taxid Bacillus velezensis (taxid 492670)
  Warning! Orphan taxid Bacilli (taxid 91061)
  Warning! Orphan taxid Salmonella enterica (taxid 28901)
  Warning! Orphan taxid Staphylococcus nepalensis (taxid 214473)
  WARNING! 186 orphan taxids (100.00% of accepted)
    and 15884 orphan sequences (100.000% of accepted)
  Assess accumulation due to "folding the tree"...
  Info: Folded TaxID Bacteroides vulgatus ATCC 8482 (taxid 435590) (Unnamed) with 2501 original seqs
  Info: Folded TaxID Cutibacterium acnes (taxid 1747) (Unnamed) with 652 original seqs
  Info: Folded TaxID Lactobacillus gasseri ATCC 33323 = JCM 1131 (taxid 324831) (Unnamed) with 617 original seqs
  Info: Folded TaxID Porphyromonas gingivalis (taxid 837) (Unnamed) with 1707 original seqs
  Info: Folded TaxID Streptococcus (taxid 1301) (Unnamed) with 547 original seqs
  Info: Folded TaxID Clostridium beijerinckii NCIMB 8052 (taxid 290402) (Unnamed) with 298 original seqs
  Info: Folded TaxID Escherichia coli (taxid 562) (Unnamed) with 491 original seqs
  Info: Folded TaxID Deinococcus radiodurans R1 (taxid 243230) (Unnamed) with 792 original seqs
  Info: Folded TaxID Streptococcus mutans (taxid 1309) (Unnamed) with 429 original seqs
  Info: Folded TaxID Staphylococcus pasteuri (taxid 45972) (Unnamed) with 73 original seqs
  Info: Folded TaxID Acinetobacter (taxid 469) (Unnamed) with 942 original seqs
  Info: Folded TaxID Neisseria meningitidis (taxid 487) (Unnamed) with 1047 original seqs
  Info: Folded TaxID Clostridium beijerinckii (taxid 1520) (Unnamed) with 206 original seqs
  Info: Folded TaxID Rhodobacter sphaeroides (taxid 1063) (Unnamed) with 523 original seqs
  Info: Folded TaxID Helicobacter pylori (taxid 210) (Unnamed) with 515 original seqs
  Info: Folded TaxID Staphylococcus aureus (taxid 1280) (Unnamed) with 300 original seqs
  Info: Folded TaxID Bifidobacterium adolescentis (taxid 1680) (Unnamed) with 338 original seqs
  Info: Folded TaxID Acinetobacter baumannii (taxid 470) (Unnamed) with 58 original seqs
  Info: Folded TaxID Rhodobacter (taxid 1060) (Unnamed) with 12 original seqs
  Info: Folded TaxID Clostridium botulinum (taxid 1491) (Unnamed) with 18 original seqs
  Info: Folded TaxID Bacteroides vulgatus (taxid 821) (Unnamed) with 840 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Typhimurium (taxid 90371) (Unnamed) with 2 original seqs
  Info: Folded TaxID Enterococcus faecalis ARO1/DG (taxid 565651) (Unnamed) with 6 original seqs
  Info: Folded TaxID Clostridium perfringens (taxid 1502) (Unnamed) with 5 original seqs
  Info: Folded TaxID Staphylococcus (taxid 1279) (Unnamed) with 339 original seqs
  Info: Folded TaxID Acinetobacter nosocomialis (taxid 106654) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus (taxid 1578) (Unnamed) with 174 original seqs
  Info: Folded TaxID Pseudomonas (taxid 286) (Unnamed) with 299 original seqs
  Info: Folded TaxID Streptococcus mutans UA159 (taxid 210007) (Unnamed) with 209 original seqs
  Info: Folded TaxID Enterobacteriaceae (taxid 543) (Unnamed) with 366 original seqs
  Info: Folded TaxID Staphylococcus epidermidis ATCC 12228 (taxid 176280) (Unnamed) with 126 original seqs
  Info: Folded TaxID Acinetobacter venetianus VE-C3 (taxid 1197884) (Unnamed) with 26 original seqs
  Info: Folded TaxID Staphylococcus succinus (taxid 61015) (Unnamed) with 16 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Bareilly str. CFSAN000191 (taxid 1173457) (Unnamed) with 6 original seqs
  Info: Folded TaxID Clostridium (taxid 1485) (Unnamed) with 241 original seqs
  Info: Folded TaxID Streptococcus agalactiae ILRI005 (taxid 1309807) (Unnamed) with 2 original seqs
  Info: Folded TaxID Bifidobacterium adolescentis ATCC 15703 (taxid 367928) (Unnamed) with 51 original seqs
  Info: Folded TaxID Staphylococcus aureus subsp. aureus (taxid 46170) (Unnamed) with 158 original seqs
  Info: Folded TaxID Bacillus cereus (taxid 1396) (Unnamed) with 3 original seqs
  Info: Folded TaxID Enterococcus faecalis (taxid 1351) (Unnamed) with 184 original seqs
  Info: Folded TaxID Bacillus (taxid 1386) (Unnamed) with 51 original seqs
  Info: Folded TaxID Staphylococcus capitis (taxid 29388) (Unnamed) with 4 original seqs
  Info: Folded TaxID Staphylococcus argenteus (taxid 985002) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli ACN001 (taxid 1311757) (Unnamed) with 22 original seqs
  Info: Folded TaxID Klebsiella pneumoniae (taxid 573) (Unnamed) with 8 original seqs
  Info: Folded TaxID Escherichia coli LY180 (taxid 1335916) (Unnamed) with 15 original seqs
  Info: Folded TaxID Escherichia coli O157:H7 str. EDL933 (taxid 155864) (Unnamed) with 55 original seqs
  Info: Folded TaxID Staphylococcus epidermidis (taxid 1282) (Unnamed) with 88 original seqs
  Info: Folded TaxID Actinomyces pacaensis (taxid 1852377) (Unnamed) with 39 original seqs
  Info: Folded TaxID Rhodobacter sphaeroides ATCC 17029 (taxid 349101) (Unnamed) with 3 original seqs
  Info: Folded TaxID Clostridium autoethanogenum DSM 10061 (taxid 1341692) (Unnamed) with 3 original seqs
  Info: Folded TaxID Enterobacterales (taxid 91347) (Unnamed) with 35 original seqs
  Info: Folded TaxID Streptococcus pyogenes (taxid 1314) (Unnamed) with 21 original seqs
  Info: Folded TaxID Escherichia coli APEC O1 (taxid 405955) (Unnamed) with 3 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa (taxid 287) (Unnamed) with 91 original seqs
  Info: Folded TaxID Clostridium sp. MF28 (taxid 1702238) (Unnamed) with 14 original seqs
  Info: Folded TaxID Staphylococcus sp. AntiMn-1 (taxid 1715860) (Unnamed) with 18 original seqs
  Info: Folded TaxID Staphylococcus caprae (taxid 29380) (Unnamed) with 11 original seqs
  Info: Folded TaxID Shigella flexneri 2002017 (taxid 591020) (Unnamed) with 14 original seqs
  Info: Folded TaxID Streptococcus agalactiae (taxid 1311) (Unnamed) with 7 original seqs
  Info: Folded TaxID Neisseria meningitidis WUE 2594 (taxid 942513) (Unnamed) with 3 original seqs
  Info: Folded TaxID Streptococcus suis T15 (taxid 1340847) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus murinus (taxid 1622) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus lindneri (taxid 53444) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus sp. FDAARGOS_522 (taxid 2420310) (Unnamed) with 10 original seqs
  Info: Folded TaxID Lactobacillus helveticus (taxid 1587) (Unnamed) with 1 original seqs
  Info: Folded TaxID Staphylococcus stepanovicii (taxid 643214) (Unnamed) with 5 original seqs
  Info: Folded TaxID Staphylococcus aureus subsp. aureus MSSA476 (taxid 282459) (Unnamed) with 1 original seqs
  Info: Folded TaxID Staphylococcus aureus 08BA02176 (taxid 1229492) (Unnamed) with 3 original seqs
  Info: Folded TaxID Streptococcus sp. FDAARGOS_520 (taxid 2420308) (Unnamed) with 2 original seqs
  Info: Folded TaxID Streptococcus pluranimalium (taxid 82348) (Unnamed) with 4 original seqs
  Info: Folded TaxID Escherichia coli O104:H4 (taxid 1038927) (Unnamed) with 7 original seqs
  Info: Folded TaxID Clostridium bornimense (taxid 1216932) (Unnamed) with 5 original seqs
  Info: Folded TaxID Acinetobacter radioresistens (taxid 40216) (Unnamed) with 2 original seqs
  Info: Folded TaxID Bacillus velezensis (taxid 492670) (Unnamed) with 5 original seqs
  Info: Folded TaxID Acinetobacter johnsonii (taxid 40214) (Unnamed) with 7 original seqs
  Info: Folded TaxID Staphylococcus condimenti (taxid 70255) (Unnamed) with 2 original seqs
  Info: Folded TaxID Staphylococcus auricularis (taxid 29379) (Unnamed) with 3 original seqs
  Info: Folded TaxID Acinetobacter calcoaceticus/baumannii complex (taxid 909768) (Unnamed) with 16 original seqs
  Info: Folded TaxID Streptococcus sp. Z15 (taxid 2173853) (Unnamed) with 1 original seqs
  Info: Folded TaxID Clostridium argentinense (taxid 29341) (Unnamed) with 6 original seqs
  Info: Folded TaxID Enterococcaceae (taxid 81852) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus parauberis NCFD 2020 (taxid 873447) (Unnamed) with 5 original seqs
  Info: Folded TaxID Pseudomonas koreensis (taxid 198620) (Unnamed) with 8 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa group (taxid 136841) (Unnamed) with 2 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa LES431 (taxid 1408272) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium beijerinckii NRRL B-598 (taxid 1428454) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas chlororaphis (taxid 587753) (Unnamed) with 2 original seqs
  Info: Folded TaxID Enterobacter bugandensis (taxid 881260) (Unnamed) with 3 original seqs
  Info: Folded TaxID Streptococcus iniae (taxid 1346) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium cellulovorans 743B (taxid 573061) (Unnamed) with 4 original seqs
  Info: Folded TaxID Staphylococcus lugdunensis (taxid 28035) (Unnamed) with 2 original seqs
  Info: Folded TaxID Enterobacter ludwigii (taxid 299767) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus anthracis str. Vollum (taxid 261591) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus cereus D17 (taxid 1454382) (Unnamed) with 2 original seqs
  Info: Folded TaxID Pseudomonas entomophila (taxid 312306) (Unnamed) with 3 original seqs
  Info: Folded TaxID Pseudomonas protegens (taxid 380021) (Unnamed) with 1 original seqs
  Info: Folded TaxID Neisseria meningitidis M7124 (taxid 1095685) (Unnamed) with 3 original seqs
  Info: Folded TaxID Staphylococcus aureus subsp. aureus NCTC 8325 (taxid 93061) (Unnamed) with 3 original seqs
  Info: Folded TaxID Streptococcus pneumoniae SPN032672 (taxid 869311) (Unnamed) with 2 original seqs
  Info: Folded TaxID Staphylococcus muscae (taxid 1294) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli O104:H4 str. 2011C-3493 (taxid 1133852) (Unnamed) with 5 original seqs
  Info: Folded TaxID Salmonella enterica (taxid 28901) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus johnsonii (taxid 33959) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium sp. CT4 (taxid 2320868) (Unnamed) with 4 original seqs
  Info: Folded TaxID Bacillus thuringiensis MC28 (taxid 1195464) (Unnamed) with 2 original seqs
  Info: Folded TaxID Staphylococcus schleiferi (taxid 1295) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lelliottia amnigena (taxid 61646) (Unnamed) with 1 original seqs
  Info: Folded TaxID Clostridium perfringens str. 13 (taxid 195102) (Unnamed) with 1 original seqs
  Info: Folded TaxID Neisseria meningitidis M04-240196 (taxid 935593) (Unnamed) with 2 original seqs
  Info: Folded TaxID Cronobacter sakazakii SP291 (taxid 956149) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium sp. JN-9 (taxid 2507159) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter sp. FDAARGOS_493 (taxid 2420300) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus anthracis (taxid 1392) (Unnamed) with 4 original seqs
  Info: Folded TaxID Morganella morganii (taxid 582) (Unnamed) with 1 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Senftenberg (taxid 28150) (Unnamed) with 2 original seqs
  Info: Folded TaxID Pseudomonas litoralis (taxid 797277) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter haemolyticus (taxid 29430) (Unnamed) with 2 original seqs
  Info: Folded TaxID Pseudomonas fluorescens (taxid 294) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium acetobutylicum (taxid 1488) (Unnamed) with 1 original seqs
  Info: Folded TaxID Clostridium baratii (taxid 1561) (Unnamed) with 1 original seqs
  Info: Folded TaxID Staphylococcus aureus subsp. aureus TCH60 (taxid 548473) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter baumannii PKAB07 (taxid 1413216) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa PA7 (taxid 381754) (Unnamed) with 3 original seqs
  Info: Folded TaxID Clostridium drakei (taxid 332101) (Unnamed) with 2 original seqs
  Info: Folded TaxID Shigella dysenteriae (taxid 622) (Unnamed) with 3 original seqs
  Info: Folded TaxID Acinetobacter bereziniae (taxid 106648) (Unnamed) with 1 original seqs
  Info: Folded TaxID Edwardsiella ictaluri (taxid 67780) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter oleivorans DR1 (taxid 436717) (Unnamed) with 2 original seqs
  Info: Folded TaxID Streptococcus intermedius C270 (taxid 862966) (Unnamed) with 3 original seqs
  Info: Folded TaxID Bacteria (taxid 2) (Unnamed) with 2 original seqs
  Info: Folded TaxID Escherichia coli UMNF18 (taxid 1050617) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus thuringiensis (taxid 1428) (Unnamed) with 2 original seqs
  Info: Folded TaxID Bacillus thuringiensis serovar finitimus YBT-020 (taxid 930170) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus reuteri (taxid 1598) (Unnamed) with 1 original seqs
  Info: Folded TaxID Shigella boydii (taxid 621) (Unnamed) with 2 original seqs
  Info: Folded TaxID Staphylococcus haemolyticus (taxid 1283) (Unnamed) with 2 original seqs
  Info: Folded TaxID Acinetobacter pittii PHEA-2 (taxid 871585) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus agalactiae GD201008-001 (taxid 1203670) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli 536 (taxid 362663) (Unnamed) with 1 original seqs
  Info: Folded TaxID Enterobacter asburiae (taxid 61645) (Unnamed) with 1 original seqs
  Info: Folded TaxID Clostridium saccharobutylicum (taxid 169679) (Unnamed) with 6 original seqs
  Info: Folded TaxID Streptococcus pyogenes MGAS15252 (taxid 798300) (Unnamed) with 2 original seqs
  Info: Folded TaxID Clostridium estertheticum subsp. estertheticum (taxid 1552) (Unnamed) with 1 original seqs
  Info: Folded TaxID Aerococcus (taxid 1375) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonadaceae (taxid 135621) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli P12b (taxid 910348) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa B136-33 (taxid 1280938) (Unnamed) with 2 original seqs
  Info: Folded TaxID Streptococcus pyogenes str. Manfredo (taxid 160491) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas aeruginosa LESB65 (taxid 1408273) (Unnamed) with 1 original seqs
  Info: Folded TaxID Staphylococcus simulans (taxid 1286) (Unnamed) with 2 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Napoli (taxid 1151001) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus delbrueckii subsp. lactis (taxid 29397) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus cereus biovar anthracis str. CI (taxid 637380) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter baumannii TYTH-1 (taxid 1100841) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus plantarum (taxid 1590) (Unnamed) with 2 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica (taxid 59201) (Unnamed) with 3 original seqs
  Info: Folded TaxID Escherichia coli APEC IMT5155 (taxid 1329907) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas xinjiangensis (taxid 487184) (Unnamed) with 1 original seqs
  Info: Folded TaxID Yersinia pestis Angola (taxid 349746) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli Nissle 1917 (taxid 316435) (Unnamed) with 1 original seqs
  Info: Folded TaxID Rhodobacter sphaeroides 2.4.1 (taxid 272943) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus halotolerans (taxid 1814128) (Unnamed) with 1 original seqs
  Info: Folded TaxID Helicobacter pylori OK113 (taxid 1248725) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter baumannii BJAB0868 (taxid 1096997) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus helveticus H10 (taxid 767462) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus cereus ATCC 10987 (taxid 222523) (Unnamed) with 1 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Montevideo str. USDA-ARS-USMARC-1901 (taxid 1454598) (Unnamed) with 1 original seqs
  Info: Folded TaxID Enterobacter cloacae (taxid 550) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter baumannii ZW85-1 (taxid 1400867) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus dysgalactiae subsp. equisimilis AC-2713 (taxid 759913) (Unnamed) with 1 original seqs
  Info: Folded TaxID Escherichia coli ABU 83972 (taxid 655817) (Unnamed) with 1 original seqs
  Info: Folded TaxID Staphylococcus nepalensis (taxid 214473) (Unnamed) with 1 original seqs
  Info: Folded TaxID Klebsiella michiganensis (taxid 1134687) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus dysgalactiae subsp. equisimilis (taxid 119602) (Unnamed) with 2 original seqs
  Info: Folded TaxID Lactobacillus sp. HSLZ-75 (taxid 1720083) (Unnamed) with 1 original seqs
  Info: Folded TaxID Candidatus Annandia adelgestsuga (taxid 1302411) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacilli (taxid 91061) (Unnamed) with 1 original seqs
  Info: Folded TaxID Pseudomonas putida (taxid 303) (Unnamed) with 1 original seqs
  Info: Folded TaxID Acinetobacter sp. SWBY1 (taxid 2079596) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus equi (taxid 1336) (Unnamed) with 1 original seqs
  Info: Folded TaxID Salmonella enterica subsp. enterica serovar Heidelberg str. B182 (taxid 1160717) (Unnamed) with 1 original seqs
  Info: Folded TaxID Lactobacillus delbrueckii subsp. delbrueckii (taxid 83684) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus cereus group (taxid 86661) (Unnamed) with 1 original seqs
  Info: Folded TaxID Streptococcus suis (taxid 1307) (Unnamed) with 1 original seqs
  Info: Folded TaxID Bacillus thuringiensis serovar kurstaki str. HD-1 (taxid 1261129) (Unnamed) with 1 original seqs
  INFO: 186 TaxIDs folded (100.00% of TAF —TaxIDs after filtering—)
  INFO: Final assigned TaxIDs: 0 (reduced to 0.00% of number of TAF)
krakenout sample VOID
Load elapsed time: 2.69 sec

Building the taxonomy multiple tree... Traceback (most recent call last):
  File "/gpfs0/home/gdlauberlab/rwr002/.local/bin/rcf", line 812, in <module>
    main()
  File "/gpfs0/home/gdlauberlab/rwr002/.local/bin/rcf", line 792, in main
    generate_krona()
  File "/gpfs0/home/gdlauberlab/rwr002/.local/bin/rcf", line 570, in generate_krona
    for sample in samples
ValueError: min() arg is an empty sequence

**Comment by wolfgangrumpf** [comment]
Additional info:  the kraken2 database was built last week; the taxdump was installed today.  So the databases *shouldn't* be out of sync.....

**Comment by khyox** [comment]
Thank you very much for the detailed bug report and for the Kraken output. The format of the Kraken file is different than expected. Could you please tell me the complete command line that you used for launching Kraken and the output of `kraken2 --version`? Thanks!

**Comment by khyox** [comment]
I think the problem could be related to the use of the `--use-names` flag in Kraken2. By chance, are you using it?

**Comment by khyox** [comment]
I have released a new version of Recentrifuge (v0.28.9) that automatically supports the Kraken2 output when the flag  '--use-names' is used. Please feel free to reopen the issue if you still have problems related to this alternative output format of Kraken2.

Your example input file is fully processed now:

```
 > rcf -n /recentrifuge/taxdump/ -k krakenout.krk

=-= /Users/martijm/local/recentrifuge/rcf =-= v0.28.9 - May 2019 =-= by Jose Manuel Martí =-=

(...)

Loading output file krakenout.krk... OK!
  Seqs read: 15_884	[25.51 Mnt]
  Seqs clas: 15_884	(0.00% unclassified)
  Seqs pass: 15_884	(0.00% rejected)
  Scores SHEL: min = 35.0, max = 3718.0, avr = 244.0
  Coverage(%): min = 0.0, max = 95.8, avr = 13.6
  Read length: min = 460 nt, max = 6.08 knt, avr = 1.60 knt
  TaxIds: by classifier = 186, by filter = 186
Building from raw data with mintaxa = 4 ...
  Check for more seqs lost ([in/ex]clude affects)... OK!
krakenout sample OK!
Load elapsed time: 3.04 sec

Building the taxonomy multiple tree... OK!
Generating final plot (krakenout.krk.rcf.html)... OK!
Generating Excel full summary (krakenout.krk.rcf.xlsx)... OK!
Total elapsed time: 00:00:24
```
![image](https://user-images.githubusercontent.com/4109713/57738719-ef24d180-76b0-11e9-9ccb-ed4b738cf8a6.png)

**Comment by wolfgangrumpf** [comment]
That's fantastic - thank you so much for the incredibly rapid response!  I'll put this into our pipeline later today and let you know if I have any other issues!

**Comment by khyox** [comment]
Thank you, Wolfgang! Sure, feel free to reopen or open another issue if further problems arise.
